Automattic/wp-calypso
# Hosting: Provide Confirmation when Copying SFTP Credentials

#### Changes proposed in this Pull Request

* Provides confirmation when copying SFTP credentials

This change is consistent with the existing pattern in Calypso - see below. I did think about moving it into the component itself, but it probably makes sense to reset the "Copied" state of other buttons once something else is copied. 

https://github.com/Automattic/wp-calypso/blob/7fe7e962ea85fab34bf7abaa846c646e9e004e8a/client/my-sites/domains/components/domain-warnings/pending-gsuite-tos-notice-dialog.jsx#L147-L155

#### Testing instructions

Click the "Copy" button in the Hosting Configuration section, and verify this changes to "Copied!". It should then reset when you use a different copy button.

<img width="658" alt="Screenshot 2021-09-21 at 14 47 25" src="https://user-images.githubusercontent.com/43215253/134183253-0de7b894-4d19-4735-a445-db88796ddb31.png">

cc @mmtr

Fixes #55316
